### PR TITLE
update access token

### PR DESCRIPTION
**Pull request by abuiquoc**

### CODE REVIEW

**Review by Pratichhya — APPROVED** [review]

File: APIs/Token.qmd [hunk 1]
```diff
@@ -57,7 +57,7 @@ curl -s -X POST https://identity.dataspace.copernicus.eu/auth/realms/CDSE/protoc
 #| eval: false
 import json
 import requests
-def get_keycloak(username: str, password: str) -> str:
+def get_access(username: str, password: str) -> str:
```
Comment (soxofaan) [inline APIs/Token.qmd:60]: or even better: `get_access_token(`

File: APIs/Token.qmd [hunk 2]
```diff
@@ -71,12 +71,12 @@ def get_keycloak(username: str, password: str) -> str:
...
     except Exception as e:
         raise Exception(
-            f"Keycloak token creation failed. Reponse from the server was: {r.json()}"
+            f"Access token creation failed. Reponse from the server was: {r.json()}"
             )
     return r.json()["access_token"]
         
 
-keycloak_token = get_keycloak("USERNAME", "PASSWORD")
+token = get_access("USERNAME", "PASSWORD")
```
Comment (soxofaan) [inline APIs/Token.qmd:79]: `access_token = get_access_token(...`

**Comment by Pratichhya** [comment]
@abuiquoc, on your behalf, I made the changes suggested by @soxofaan, i.e., renamed the variable as access_token and the function as get_access_token. Please let me know if us have any comments.

**Comment by abuiquoc** [comment]
No comments, We can publish changes.

File: APIs/Token.qmd (hunk 1 above)
Comment (Pratichhya) [reply APIs/Token.qmd:60]: updated

File: APIs/Token.qmd (hunk 2 above)
Comment (Pratichhya) [reply APIs/Token.qmd:79]: updated

**Pull request merged**